src/styles/components/_component-grid.scss

```diff
@@ -50,24 +109,93 @@ $breakpoint-tablet: 820px;
       ". max-scott .";
 
     &__item {
+      border: 3px solid black;
+      text-align: center;
+      align-items: center;
+
       &--accordion {
+        position: relative;
         grid-area: accordion;
+
+        &::before {
+          content: "";
+          position: absolute;
+          top: -11%;
+          left: -7%;
+          rotate: -15deg;
+          width: 40%;
+          height: 20%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_line.png");
+          z-index: 1;
+        }
       }
 
       &--dialog {
+        position: relative;
         grid-area: dialog;
+
+        &::before {
+          content: "";
+          position: absolute;
+          bottom: -3%;
+          right: -3%;
+          width: 10%;
+          height: 10%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_o.png");
+          z-index: 1;
+        }
       }
 
       &--disclosure {
+        position: relative;
         grid-area: disclosure;
+
+        &::before {
+          content: "";
+          position: absolute;
+          top: -5%;
+          right: -3%;
+          width: 20%;
+          height: 20%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_diamond.png");
+          z-index: 1;
+        }
       }
 
       &--tabs {
+        position: relative;
         grid-area: tabs;
+
+        &::before {
+          content: "";
+          position: absolute;
+          bottom: -15%;
+          left: -3%;
+          width: 30%;
```

Comment:
It looks like this image container is taking up bit more space than intended, we should be able to reduce the width here to at least 15%

Reply:
Good catch--the diamonds were also a bit larger than they should have been! I adjust both doodles in c8a67b9.